pkg/controlplane/authz.go

```diff
@@ -79,12 +79,20 @@ func (cp *Instance) AuthorizeEgress(req *EgressAuthorizationRequest) (*EgressAut
 		return nil, fmt.Errorf("no bindings found for import '%s'", req.Import)
 	}
 
-	// TODO: get k8s attributes using cp.kubeClient
 	connReq := eventmanager.ConnectionRequestAttr{DstService: req.Import, Direction: eventmanager.Outgoing}
+	srcLabels := cp.platform.GetLabelsFromIP(req.IP)
+
+	if srcLabels != nil {
```

Comment:
no need to check for nil value. [The nil map supports queries](https://go.dev/ref/spec#Index_expressions), and will return the same results as an empty map.
It's only insertions to the nil map that panic.

Reply:
Done